# Hindi translation for update-manager
# Copyright (c) (c) 2006 Canonical Ltd, and Rosetta Contributors 2006
# This file is distributed under the same license as the update-manager package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2006.
#
#: ../DistUpgrade/DistUpgradeController.py:1075
msgid ""
msgstr ""
"Project-Id-Version: update-manager\n"
"Report-Msgid-Bugs-To: sebastian.heinlein@web.de\n"
"POT-Creation-Date: 2020-08-03 09:26-0700\n"
"PO-Revision-Date: 2013-05-22 11:00+0000\n"
"Last-Translator: Manish Kumar <manishku86@yahoo.co.in>\n"
"Language-Team: Hindi <hi@li.org>\n"
"Language: hi\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Launchpad-Export-Date: 2019-11-15 04:47+0000\n"
"X-Generator: Launchpad (build c597c3229eb023b1e626162d5947141bf7befb13)\n"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:215 ../DistUpgrade/distro.py:446
#, python-format
msgid "Server for %s"
msgstr "%s का सर्वर"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:233 ../DistUpgrade/distro.py:239
#: ../DistUpgrade/distro.py:255
msgid "Main server"
msgstr "मुख्य सर्वर"

#: ../DistUpgrade/distro.py:259
msgid "Custom servers"
msgstr "विनिर्मित सर्वर"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:151
msgid "Could not calculate sources.list entry"
msgstr "sources.list प्रविष्टि क़ी गणना नहीँ हो सकी"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:260
msgid ""
"Unable to locate any package files, perhaps this is not a Ubuntu Disc or the "
"wrong architecture?"
msgstr "पैकेज संचिकाएँ नहीं मिलीं, संभवत: यह उबुंतू डिस्क नहीं है या स्थापत्य अनुपयुक्त है."

#: ../DistUpgrade/DistUpgradeAptCdrom.py:310
msgid "Failed to add the CD"
msgstr "सी.डी. योजन असफल रहा."

#: ../DistUpgrade/DistUpgradeAptCdrom.py:311
#, python-format
msgid ""
"There was a error adding the CD, the upgrade will abort. Please report this "
"as a bug if this is a valid Ubuntu CD.\n"
"\n"
"The error message was:\n"
"'%s'"
msgstr ""
"सीoडीo योजन में त्रुटि हुई, उन्नयन की प्रक्रिया बन्द होगी | यदि यह वैध उबुंतू सीoडo | है "
"तो कृपया इस घटना को बग रिपोर्ट करें |\n"
"त्रुटि संदेश था:\n"
"'%s'"

#: ../DistUpgrade/DistUpgradeCache.py:182
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "अवांछित स्थिति के पैकेज हटाएँ."
msgstr[1] "अवांछित पैकेजों को हटाएँ"

#: ../DistUpgrade/DistUpgradeCache.py:185
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"पैकेज '%s' असंगत स्थिति में है और इसके पुनर्स्थापन की आवश्यकता है, किंतु इसके लिए कोई आलेख "
"नहीं मिला. क्या जारी रखने के लिए आप इस पैकेज को हटाना चाहेंगें?"
msgstr[1] ""
"एकाधिक पैकेज '%s' असंगत स्थिति में हैं और इनके पुनर्स्थापन की आवश्यकता है, किंतु इनके लिए आलेख "
"नहीं मिले. क्या जारी रखने के लिए आप इन पैकेज को हटाना चाहेंगें?"

#. FIXME: not ideal error message, but we just reuse a
#. existing one here to avoid a new string
#: ../DistUpgrade/DistUpgradeCache.py:301
msgid "The server may be overloaded"
msgstr "हो सकता है की सर्वर अतिभारित है"

#: ../DistUpgrade/DistUpgradeCache.py:414
msgid "Broken packages"
msgstr "अवांछित पैकेज"

#: ../DistUpgrade/DistUpgradeCache.py:415
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"आपके सिस्टम में अवांछित पैकेज है जो इस सॉफ्टवेयर के साथ ठीक नहीं किए जा सकते है | कृपया उन्हें "
"आगे बढ़ने से पहले सिनेपटिक या apt-get के साथ ठीक करे  |"

#: ../DistUpgrade/DistUpgradeCache.py:660
#, fuzzy, python-format
msgid "Broken packages after upgrade: %s"
msgstr "अवांछित पैकेज"

#. the most likely problem is the 3rd party pkgs so don't address
#. foreignPkgs and devRelease being True
#: ../DistUpgrade/DistUpgradeCache.py:669
msgid ""
"An unresolvable problem occurred while calculating the upgrade.\n"
"\n"
" "
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:672
msgid ""
"This was likely caused by:\n"
" * Unofficial software packages not provided by Ubuntu\n"
"Please use the tool 'ppa-purge' from the ppa-purge \n"
"package to remove software from a Launchpad PPA and \n"
"try the upgrade again.\n"
"\n"
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:680
msgid ""
"This was caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
"This is most likely a transient problem, \n"
"please try again later.\n"
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:687
msgid "This is most likely a transient problem, please try again later."
msgstr "बहुत सम्भव है कि यह अस्थायी समस्या है, कृपया बाद में पुनः प्रयास करें |"

#: ../DistUpgrade/DistUpgradeCache.py:690
msgid ""
"If none of this applies, then please report this bug using the command "
"'ubuntu-bug ubuntu-release-upgrader-core' in a terminal. "
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:692
msgid ""
"If you want to investigate this yourself the log files in '/var/log/dist-"
"upgrade' will contain details about the upgrade. Specifically, look at 'main."
"log' and 'apt.log'."
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:698
msgid "Could not calculate the upgrade"
msgstr "नवीनीकरण की गणना नहीं की जा सकी"

#: ../DistUpgrade/DistUpgradeCache.py:757
msgid "Error authenticating some packages"
msgstr "कुछ पैकेज के सत्यापन में त्रुटि"

#: ../DistUpgrade/DistUpgradeCache.py:758
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"यह सम्भव है कि कुछ पैकेज के प्रामाणित करना सम्भव नहीं है। यह नेटवर्क का अस्थायी समस्या हो "
"सकता है | आप बाद में पुनः प्रयास कर सकते है | प्रमाणित पैकेज सूची के लिए नीचे देखें |"

#: ../DistUpgrade/DistUpgradeCache.py:779
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal blacklist."
msgstr ""
"पैकेज '%s' को हटाने के लिए चिह्नित किया गया है लेकिन यह हटाए जाने वाली कालीसूची में है |"

#: ../DistUpgrade/DistUpgradeCache.py:785
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "अत्यावश्यक पैकेज '%s' को हटाने हेतु चिह्नित किया गया है |"

#: ../DistUpgrade/DistUpgradeCache.py:794
#, python-format
msgid "Trying to install blacklisted version '%s'"
msgstr "कालीसुचित वर्जन  '%s' संस्थापित करने की कोशिश कर रहा है"

#: ../DistUpgrade/DistUpgradeCache.py:916
#, python-format
msgid "Can't install '%s'"
msgstr "%s संस्थापित नहीं हो सका"

#: ../DistUpgrade/DistUpgradeCache.py:917
msgid ""
"It was impossible to install a required package. Please report this as a bug "
"using 'ubuntu-bug ubuntu-release-upgrader-core' in a terminal."
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:928
msgid "Can't guess meta-package"
msgstr "मेटा-पैकेज का अनुमान न करें"

#: ../DistUpgrade/DistUpgradeCache.py:929
#, python-format
msgid ""
"Your system does not contain a %s or %s package and it was not possible to "
"detect which version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:113
msgid "Reading cache"
msgstr "कैची पढ़ रहा है"

#: ../DistUpgrade/DistUpgradeController.py:225
msgid "Unable to get exclusive lock"
msgstr "अपवर्जित ताला प्राप्त करने में अक्षम"

#: ../DistUpgrade/DistUpgradeController.py:226
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"इसका मतलब यह हुआ कि दूसरा पैकेज प्रबंधक अनुप्रयोग (जैसे apt-get या aptitude) पहले से चल "
"रहा है |"

#: ../DistUpgrade/DistUpgradeController.py:284
msgid "Upgrading over remote connection not supported"
msgstr "दूरस्थ संयोजन द्वारा अद्यतन समर्थित नहीं है"

#: ../DistUpgrade/DistUpgradeController.py:285
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. Please try a text mode upgrade with 'do-release-"
"upgrade'.\n"
"\n"
"The upgrade will abort now. Please try without ssh."
msgstr ""
"आप अग्रछोड़ पर दूरस्थ ssh संयोजन द्वारा अद्यतन चला रहे हैं जो इसे समर्थित नहीं करता है | "
"कृपया पाठ पद्धति से 'do-release-upgrade' द्वारा अद्यतन करें |\n"
"\n"
"अद्यतन छोड़ रहा है. कृपया बिना ssh के प्रयास करें |"

#: ../DistUpgrade/DistUpgradeController.py:299
msgid "Continue running under SSH?"
msgstr "SSH के अंतर्गत आगे चलाएं?"

#: ../DistUpgrade/DistUpgradeController.py:300
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, an additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"यह सत्र ssh के अंतर्गत चल रहा है. सलाह दी जाती है कि वर्तमान में ssh द्वारा अद्यतन न करें "
"क्योंकि असफल होने की स्थिति में पुनःप्राप्ति संभव नहीं है |\n"
"यदि आप जारी रखते है, तो अतिरिक्त ssh डेमन '%s' पोर्ट पर आरंभ हो जाएगा |"

#: ../DistUpgrade/DistUpgradeController.py:314
msgid "Starting additional sshd"
msgstr "अतिरिक्त sshd आरंभ हो रहा है"

#: ../DistUpgrade/DistUpgradeController.py:315
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"असफल होने की स्थिति में समुत्थान हेतु, एक अतिरिक्त sshd का आरंभ पोर्ट '%s' पर हो जाएगा "
"| यदि चल रहे ssh में कुछ गड़बड़ी होती है तो आप अतिरिक्त एक और के साथ संयोजित रहेगें |\n"

#: ../DistUpgrade/DistUpgradeController.py:323
#, python-format
msgid ""
"If you run a firewall, you may need to temporarily open this port. As this "
"is potentially dangerous it's not done automatically. You can open the port "
"with e.g.:\n"
"'%s'"
msgstr ""
"यदि आप फ़ायरवॉल चला रहे हैं, तो आप इस पोर्ट को अस्थायी रुप से खोलना होगा | यह "
"स्वचालित रुप से नहीं हुआ क्योंकि ऐसा करना खतरनाक हो सकता है | आप पोर्ट खोल सकते है उदा. "
"के लिए:\n"
"'%s'"

#: ../DistUpgrade/DistUpgradeController.py:401
#: ../DistUpgrade/DistUpgradeController.py:425
msgid "Can not upgrade"
msgstr "अद्यतन नहीं हो सका"

#: ../DistUpgrade/DistUpgradeController.py:402
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr "अद्यतन '%s' से '%s' तक इस औजार द्वारा समर्थित नहीं है |"

#: ../DistUpgrade/DistUpgradeController.py:426
msgid ""
"Your python3 install is corrupted. Please fix the '/usr/bin/python3' symlink."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:455
#, python-format
msgid "Can not write to '%s'"
msgstr "'%s' में नहीं लिख सकते"

#: ../DistUpgrade/DistUpgradeController.py:456
#, python-format
msgid ""
"Its not possible to write to the system directory '%s' on your system. The "
"upgrade can not continue.\n"
"Please make sure that the system directory is writable."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:467
msgid "Include latest updates from the Internet?"
msgstr "अंतर्जाल से नवीनतम अद्यतन समाहित करें?"

#: ../DistUpgrade/DistUpgradeController.py:468
msgid ""
"The upgrade system can use the internet to automatically download the latest "
"updates and install them during the upgrade.  If you have a network "
"connection this is highly recommended.\n"
"\n"
"The upgrade will take longer, but when it is complete, your system will be "
"fully up to date.  You can choose not to do this, but you should install the "
"latest updates soon after upgrading.\n"
"If you answer 'no' here, the network is not used at all."
msgstr ""
"अद्यतन तंत्र अंतर्जाल का उपयोग कर स्वतः ही नवीनतम अद्यतन को डाउनलोड कर लेगा तथा "
"अद्यतन के दौरान उसे संस्थापित कर देगा यदि आपके पास अंतर्जाल संयोजन है इसकी पुरजोर "
"सिफारिश की जाती है\n"
"\n"
"अद्यतन लंबा समय लेगा लेकिन जब यह पूर्ण हो जाएगा, आपका तंत्र पूर्णतः अद्यतन रहेगा. आप "
"चाहे. ऐसा न हो. चुन सकते है लेकिन आप अद्यतन करने के पश्चात यथाशीध्र नवीनतम अद्यतन को "
"संस्थापित करने हेतु चुन सकते हैं |\n"
"यदि आपका जबाब यहाँ पर 'नहीं' होता है तो संजाल को पूर्णतः उपयोग नहीं होता |"

#: ../DistUpgrade/DistUpgradeController.py:646
#, python-format
msgid "Not for humans during development stage of release %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:747
#, python-format
msgid "disabled on upgrade to %s"
msgstr "%s को अद्यतन हेतु अशक्त करें"

#: ../DistUpgrade/DistUpgradeController.py:804
msgid "No valid sources.list entry found"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:805
#, python-format
msgid ""
"While scanning your repository information no entry about %s could be "
"found.\n"
"\n"
"An upgrade might not succeed.\n"
"\n"
"Do you want to continue anyway?"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:817
msgid "No valid mirror found"
msgstr "वैध मिरर नहीं पाया गया"

#: ../DistUpgrade/DistUpgradeController.py:818
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found. This can happen if you run an internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'No' the upgrade will cancel."
msgstr ""

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:838
msgid "Generate default sources?"
msgstr "डिफाल्ट स्त्रोत बनाएं?"

#: ../DistUpgrade/DistUpgradeController.py:839
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""
"आपके 'sources.list' की जाँच करने के पश्चात कोई भी वैध प्रविष्टि '%s' हेतु नहीं पाई गई "
"|\n"
"\n"
"\n"
"क्या '%s' हेतु डिफाल्ट प्रविष्टि जोड़ा जाए? यदि आप 'नहीं' चूनते है तो अद्यतन निरस्त हो "
"जाएगा|"

#: ../DistUpgrade/DistUpgradeController.py:874
#: ../DistUpgrade/DistUpgradeController.py:880
msgid "Repository information invalid"
msgstr "भण्डार सूचना अवैध है"

#: ../DistUpgrade/DistUpgradeController.py:875
msgid ""
"Upgrading the repository information resulted in a invalid file so a bug "
"reporting process is being started."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:881
msgid ""
"Upgrading the repository information resulted in a invalid file. To report a "
"bug install apport and then execute 'apport-bug ubuntu-release-upgrader'."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:890
msgid "Third party sources disabled"
msgstr "तृतीय पक्ष स्त्रोत अशक्त है"

#: ../DistUpgrade/DistUpgradeController.py:891
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"आपके source.list कुछ तृतीय पक्ष प्रविष्टि अशक्त है | आप इसे अद्यतन के पश्चात अपने पैकेज "
"प्रबंधक द्वारा या 'software-properties' उपकरण द्वारा पुनःसशक्त कर सकते हैं |"

#: ../DistUpgrade/DistUpgradeController.py:933
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "पैकेज असंगत स्थिति में है"
msgstr[1] "पैकेज असंगत स्थिति में है"

#: ../DistUpgrade/DistUpgradeController.py:936
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"'%s' पैकेज असंगत स्थिति में है तथा पुनःसंस्थापित करने की जरुरत है, लेकिन इसके लिए कोई अभिलेख "
"प्राप्त नहीं हुआ है | कृपया पैकेज को हस्तगत (मैनुअली) पुनःसंस्थापित करें या इसे तंत्र से हटा दें |"
msgstr[1] ""
"'%s' पैकेज असंगत स्थिति में है तथा पुनःसंस्थापित करने की जरुरत है, लेकिन इनके लिए कोई अभिलेख "
"प्राप्त नहीं हुआ है | कृपया पैकेज को हस्तगत (मैनुअली) पुनःसंस्थापित करें या इन सबको तंत्र से "
"हटा दें |"

#: ../DistUpgrade/DistUpgradeController.py:999
msgid "Error during update"
msgstr "अद्यतन के दौरान त्रुटि"

#: ../DistUpgrade/DistUpgradeController.py:1000
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"अद्यतन के दौरान एक त्रुटि पाई गयी | यह समान्यतः संजाल समस्या के कारण हो सकता है, कृपया "
"संजाल संयोजन की जाँच करें तथा पुनः प्रयास करें |"

#: ../DistUpgrade/DistUpgradeController.py:1043
#: ../DistUpgrade/DistUpgradeController.py:1048
msgid "EFI System Partition (ESP) not usable"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1044
msgid ""
"Your EFI System Partition (ESP) is not mounted at /boot/efi. Please ensure "
"that it is properly configured and try again."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1049
msgid ""
"The EFI System Partition (ESP) mounted at /boot/efi is not writable. Please "
"mount this partition read-write and try again."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1056
msgid "Not enough free disk space"
msgstr "डिस्क प्रयाप्त खाली जगह नहीं है"

#. TRANSLATORS: you can change the order of the sentence,
#. make sure to keep all {str_*} string untranslated.
#: ../DistUpgrade/DistUpgradeController.py:1059
#, python-brace-format
msgid ""
"The upgrade has aborted. The upgrade needs a total of {str_total} free space "
"on disk '{str_dir}'. Please free at least an additional {str_needed} of disk "
"space on '{str_dir}'. {str_remedy}"
msgstr ""

#. specific ways to resolve lack of free space
#: ../DistUpgrade/DistUpgradeController.py:1064
msgid ""
"Remove temporary packages of former installations using 'sudo apt clean'."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1066
msgid ""
"You can remove old kernels using 'sudo apt autoremove' and you could also "
"set COMPRESS=xz in /etc/initramfs-tools/initramfs.conf to reduce the size of "
"your initramfs."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1071
msgid ""
"Empty your trash and remove temporary packages of former installations using "
"'sudo apt-get clean'."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1074
msgid "Reboot to clean up files in /tmp."
msgstr ""

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:1113
#: ../DistUpgrade/DistUpgradeController.py:1989
#: ../DistUpgrade/DistUpgradeController.py:2039
msgid "Calculating the changes"
msgstr "परिवर्तन की गणना कर रहा है"

#. ask the user
#: ../DistUpgrade/DistUpgradeController.py:1157
msgid "Do you want to start the upgrade?"
msgstr "आप अद्यतन आरंभ करना चाहते हैं?"

#: ../DistUpgrade/DistUpgradeController.py:1176
#, python-format
msgid ""
"Livepatch security updates are not available for Ubuntu %s. If you upgrade, "
"Livepatch will turn off."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1248
msgid "Upgrade canceled"
msgstr "अद्यतन निरस्त"

#: ../DistUpgrade/DistUpgradeController.py:1249
msgid ""
"The upgrade will cancel now and the original system state will be restored. "
"You can resume the upgrade at a later time."
msgstr ""
"यह उन्नयन अब निरस्त हो जाएगा तथा मूल तंत्र स्थिति लौट आएगा | आप उन्नयन को बाद में कभी "
"पुनःशुरु कर सकते हैं |"

#: ../DistUpgrade/DistUpgradeController.py:1255
#: ../DistUpgrade/DistUpgradeController.py:1400
msgid "Could not download the upgrades"
msgstr "अद्यतन डाउनलोड नहीं कर सका"

#: ../DistUpgrade/DistUpgradeController.py:1256
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far have been kept."
msgstr ""

#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:1349
#: ../DistUpgrade/DistUpgradeController.py:1388
#: ../DistUpgrade/DistUpgradeController.py:1504
msgid "Error during commit"
msgstr "बचनबद्धता के दौरान त्रुटि"

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1351
#: ../DistUpgrade/DistUpgradeController.py:1390
#: ../DistUpgrade/DistUpgradeController.py:1543
msgid "Restoring original system state"
msgstr "मूल तंत्र स्थिति को पुनःसंगृहित कर रहा है"

#: ../DistUpgrade/DistUpgradeController.py:1352
#: ../DistUpgrade/DistUpgradeController.py:1367
#: ../DistUpgrade/DistUpgradeController.py:1391
msgid "Could not install the upgrades"
msgstr "अद्यतन को संस्थापित नहीं कर सका"

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:1357
msgid ""
"The upgrade has aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"उन्नयन निष्फल हो गया. आपका तंत्र अनुपयोगी स्थिति में आ सकता है | एक समुत्थान अब चलेगा "
"(dpkg --configure -a) |"

#: ../DistUpgrade/DistUpgradeController.py:1362
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug in a browser at http://bugs.launchpad.net/ubuntu/"
"+source/ubuntu-release-upgrader/+filebug and attach the files in /var/log/"
"dist-upgrade/ to the bug report.\n"
"%s"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1401
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr ""
"उन्नयन निष्फल हो गया | कृपया अपने अंतर्जाल संयोजन या संस्थापना मिडिया की जाँच कर लें तथा "
"पुनः प्रयास करें | "

#: ../DistUpgrade/DistUpgradeController.py:1429
msgid "Searching for obsolete software"
msgstr "अप्रचलित साफ्टवेयर को खोज रहा है"

#: ../DistUpgrade/DistUpgradeController.py:1492
msgid "Remove obsolete packages?"
msgstr "अप्रचलित पैकेज को हटाएं?"

#: ../DistUpgrade/DistUpgradeController.py:1493
#: ../data/gtkbuilder/DistUpgrade.ui.h:8
msgid "_Keep"
msgstr "बनाए रखें (_K)"

#: ../DistUpgrade/DistUpgradeController.py:1493
msgid "_Remove"
msgstr "हटाएँ (_R)"

#: ../DistUpgrade/DistUpgradeController.py:1505
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr ""
"सफाई के दौरान समस्या उत्पन्न हो गई| अधिक जानकारी के लिए कृपया निम्न सूचना को देखें | "

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1581
msgid "Required depends is not installed"
msgstr "आवश्यक निर्भर संस्थापित नहीं है"

#: ../DistUpgrade/DistUpgradeController.py:1582
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "आवश्यक निर्भरता '%s' संस्थापित नहीं है. "

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1849
#: ../DistUpgrade/DistUpgradeController.py:1939
msgid "Checking package manager"
msgstr "पैकेज प्रबंधक की जाँच करें"

#: ../DistUpgrade/DistUpgradeController.py:1855
#: ../DistUpgrade/DistUpgradeController.py:1861
msgid "Preparing the upgrade failed"
msgstr "अद्यतन की तैयारी असफल"

#: ../DistUpgrade/DistUpgradeController.py:1856
msgid ""
"Preparing the system for the upgrade failed so a bug reporting process is "
"being started."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1862
msgid ""
"Preparing the system for the upgrade failed. To report a bug install apport "
"and then execute 'apport-bug ubuntu-release-upgrader'."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1879
#: ../DistUpgrade/DistUpgradeController.py:1889
msgid "Getting upgrade prerequisites failed"
msgstr "पूर्वापेक्षित अद्यतन को प्राप्त करने में असफल"

#: ../DistUpgrade/DistUpgradeController.py:1880
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Additionally, a bug reporting process is being started."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1890
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"To report a bug install apport and then execute 'apport-bug ubuntu-release-"
"upgrader'."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1923
msgid "Updating repository information"
msgstr "भण्डार सूचना का अद्यतन हो रहा है"

#: ../DistUpgrade/DistUpgradeController.py:1930
msgid "Failed to add the cdrom"
msgstr "सीडीरोम जोड़ने में असफल"

#: ../DistUpgrade/DistUpgradeController.py:1931
msgid "Sorry, adding the cdrom was not successful."
msgstr "खेद है, सीडीरोम सफलतापूर्वक नहीं जोड़ा जा सका |"

#: ../DistUpgrade/DistUpgradeController.py:1966
msgid "Invalid package information"
msgstr "पैकेज सूचना अवैध"

#: ../DistUpgrade/DistUpgradeController.py:1967
#, python-format
msgid ""
"After updating your package information, the essential package '%s' could "
"not be located. This may be because you have no official mirrors listed in "
"your software sources, or because of excessive load on the mirror you are "
"using. See /etc/apt/sources.list for the current list of configured software "
"sources.\n"
"In the case of an overloaded mirror, you may want to try the upgrade again "
"later."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1996
#: ../DistUpgrade/DistUpgradeController.py:2095
msgid "Fetching"
msgstr "आकर्षित कर रहा है"

#: ../DistUpgrade/DistUpgradeController.py:2003
#: ../DistUpgrade/DistUpgradeController.py:2024
#: ../DistUpgrade/DistUpgradeController.py:2045
#: ../DistUpgrade/DistUpgradeController.py:2099
msgid "Upgrading"
msgstr "अद्यतन कर रहा है"

#: ../DistUpgrade/DistUpgradeController.py:2005
msgid "Upgrade infeasible"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:2006
msgid ""
"The upgrade could not be completed, there were errors during the upgrade "
"process."
msgstr ""

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:2027
msgid "Upgrade incomplete"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:2028
msgid ""
"The upgrade has partially completed but there were errors during the upgrade "
"process."
msgstr ""

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:2050
#: ../DistUpgrade/DistUpgradeController.py:2101
#: ../DistUpgrade/DistUpgradeController.py:2107
#: ../DistUpgrade/DistUpgradeController.py:2118
msgid "Upgrade complete"
msgstr "अद्यतन पूर्ण"

#: ../DistUpgrade/DistUpgradeController.py:2051
#: ../DistUpgrade/DistUpgradeController.py:2102
#: ../DistUpgrade/DistUpgradeController.py:2108
msgid ""
"The upgrade has completed but there were errors during the upgrade process."
msgstr "उन्नयन पुरा हुआ लेकिम उन्नयन प्रक्रिया के दौरान त्रुटि हुई |"

#: ../DistUpgrade/DistUpgradeController.py:2070
msgid "System upgrade is complete."
msgstr "तंत्र का अद्यतन पूर्ण |"

#: ../DistUpgrade/DistUpgradeController.py:2119
msgid "The partial upgrade was completed."
msgstr "आंशिक अद्यतन पूर्ण हुआ |"

#: ../DistUpgrade/DistUpgradeController.py:2147
msgid "Lock screen disabled"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:2148
msgid ""
"Your lock screen has been disabled and will remain disabled until you reboot."
msgstr ""

#: ../DistUpgrade/DistUpgradeFetcher.py:107
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:170
msgid "Could not find the release notes"
msgstr "प्रकाशन सूचना नहीं पाया गया"

#: ../DistUpgrade/DistUpgradeFetcher.py:108
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:171
msgid "The server may be overloaded. "
msgstr "सर्वर अतिभारित है. "

#: ../DistUpgrade/DistUpgradeFetcher.py:120
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:175
msgid "Could not download the release notes"
msgstr "प्रकाशन सूचना डाउनलोड नहीं कर सका"

#: ../DistUpgrade/DistUpgradeFetcher.py:121
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:176
msgid "Please check your internet connection."
msgstr "अंतर्जाल संयोजन की जाँच करें."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:73
#, python-format
msgid "authenticate '%(file)s' against '%(signature)s' "
msgstr ""

#: ../DistUpgrade/DistUpgradeFetcherCore.py:97
#, python-format
msgid "extracting '%s'"
msgstr "'% s' निकला जा रहा है"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:118
#: ../DistUpgrade/DistUpgradeFetcherCore.py:119
msgid "Could not run the upgrade tool"
msgstr "अद्यतन उपकरण को नहीं चला सका"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:120
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug "
"using the command 'ubuntu-bug ubuntu-release-upgrader-core'."
msgstr ""

#: ../DistUpgrade/DistUpgradeFetcherCore.py:197
msgid "Upgrade tool signature"
msgstr "अद्यतन उपकरण हस्ताक्षर"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:204
msgid "Upgrade tool"
msgstr "अद्यतन उपकरण"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:238
msgid "Failed to fetch"
msgstr "मिलान करने में असफल"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:239
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr "अद्यतन का मिलान असफल. यह संजाल समस्या के कारण हो सकता है "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:243
msgid "Authentication failed"
msgstr "प्रमाणीकरण असफल"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:244
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr ""
"अद्यतन का प्रमाणिकरण निष्फल. यह आपके संजाल या सर्वर के साथ समस्या के काऱण हो सकता है. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:249
msgid "Failed to extract"
msgstr "अवतरण करने में असफल"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:250
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr ""
"अद्यतन का अवतरण करने में असफल. यह संजाल या सर्वर के साथ समस्या के कारण हो सकता है. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:255
msgid "Verification failed"
msgstr "प्रमाणीकरण असफल"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:256
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr ""
"अद्यतन का प्रमाणीकरण असफल. यह संजाल या सर्वर के साथ समस्या के कारण हो सकता है. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:270
#: ../DistUpgrade/DistUpgradeFetcherCore.py:276
msgid "Can not run the upgrade"
msgstr "अद्यतन नहीं हो सकता"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:271
msgid ""
"This usually is caused by a system where /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""

#: ../DistUpgrade/DistUpgradeFetcherCore.py:277
#, python-format
msgid "The error message is '%s'."
msgstr "त्रुटि संदेश '%s' है."

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:151
msgid "&Upgrade"
msgstr ""

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:154
#: ../DistUpgrade/DistUpgradeViewKDE.py:715
msgid "&Cancel"
msgstr ""

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:156
#: ../data/gtkbuilder/ReleaseNotes.ui.h:1
msgid "Release Notes"
msgstr "प्रकाशन सूचना"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:191
msgid "Upgrade"
msgstr "उन्नत बनाएं"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:207
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:222
msgid "Downloading additional package files..."
msgstr "अतिरिक्त पैकेज संचिका को डाउनलोड कर रहा है..."

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:224
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "संचिका %s है %s में से %sB/s की दर से"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:228
#, python-format
msgid "File %s of %s"
msgstr "संचिका %s है %s में से"

#. print("mediaChange %s %s" % (medium, drive))
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:235
#: ../DistUpgrade/DistUpgradeViewGtk3.py:125
#: ../DistUpgrade/DistUpgradeViewKDE.py:302
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "कृपया '%s' को ड्राइव '%s' में डालें"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:237
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:242
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:243
#: ../DistUpgrade/DistUpgradeViewKDE.py:303
msgid "Media Change"
msgstr "मिडिया परिवर्तन"

#: ../DistUpgrade/DistUpgradeQuirks.py:227
msgid "Your graphics hardware may not be fully supported in Ubuntu 14.04."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:229
msgid ""
"Running the 'unity' desktop environment is not fully supported by your "
"graphics hardware. You will maybe end up in a very slow environment after "
"the upgrade. Our advice is to keep the LTS version for now. For more "
"information see https://wiki.ubuntu.com/X/Bugs/"
"UpdateManagerWarningForUnity3D Do you still want to continue with the "
"upgrade?"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:253
msgid "Your graphics hardware may not be fully supported in Ubuntu 12.04 LTS."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:255
msgid ""
"The support in Ubuntu 12.04 LTS for your Intel graphics hardware is limited "
"and you may encounter problems after the upgrade. For more information see "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx Do you want to "
"continue with the upgrade?"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:275
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr ""
"अद्यतन डेक्सटाप तथा खेल निष्पादन तथा अन्य चित्रादि वाले कार्यक्रम को के प्रभाव को कम कर "
"देगा."

#: ../DistUpgrade/DistUpgradeQuirks.py:279
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 10.04 "
"LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"यह कंप्यूटर वर्तमान में एएमडी का 'fglrx' चित्रादि ड्राइवर इस्तेमाल कर रहा है. उबून्टू "
"10.04 LTS में आपके हार्डवेयर के साथ कार्य करने हेतु इसका कोई भी संस्करण उपलब्ध नहीं है.\n"
"\n"
"क्या आप आगे बढ़ना चाहते हैं?"

#: ../DistUpgrade/DistUpgradeQuirks.py:318
msgid "Sorry, no more upgrades for this system"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:319
#, python-format
msgid ""
"There will not be any further Ubuntu releases for this system's 'i386' "
"architecture.\n"
"\n"
"Updates for Ubuntu %s will continue until %s."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:334
msgid "No ARMv6 CPU"
msgstr "ARMv6 सीपीयु नहीं है"

#: ../DistUpgrade/DistUpgradeQuirks.py:335
msgid ""
"Your system uses an ARM CPU that is older than the ARMv6 architecture. All "
"packages in karmic were built with optimizations requiring ARMv6 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"आपका तंत्र ARM सीपीयु का उपयोग कर रहा है जो ARMv6 संरचना से ज्यादा पुराना है. "
"'कार्मिक' के सभी पैकेज इस आशावादिता के साथ बनाए गए है कि न्यूनतम संरचना ARMv6 की "
"आवश्यकता होगी. इन हार्डवेयर के साथ उबुन्टू के नए प्रकाशन द्वारा अपने तंत्र को उन्नत करना "
"संभव नहीं है."

#: ../DistUpgrade/DistUpgradeQuirks.py:355
msgid "No init available"
msgstr "init उपलब्ध नहीं है"

#: ../DistUpgrade/DistUpgradeQuirks.py:356
msgid ""
"Your system appears to be a virtualised environment without an init daemon, "
"e.g. Linux-VServer. Ubuntu 10.04 LTS cannot function within this type of "
"environment, requiring an update to your virtual machine configuration "
"first.\n"
"\n"
"Are you sure you want to continue?"
msgstr ""
"आपका तंत्र आभासी वातावरण में बिना किसी inti डेमन अर्थात लाईनेक्स-Vसर्वर के कार्य कर रहा "
"है. उबुन्टू 10.04 LTS इस प्रकार के वातावरण में कार्य नहीं कर सकता, पहले अपने आभासी मशीन "
"को अद्यतन करने की जरुरत है."

#: ../DistUpgrade/DistUpgradeQuirks.py:469
msgid "Connection to the Snap Store failed"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:470
msgid ""
"You have the package lxd installed but your system is unable to reach the "
"Snap Store. lxd is now provided via a snap and the release upgrade will fail "
"if snapd is not functional. Please make sure you're connected to the "
"Internet and update any firewall or proxy settings as needed so that you can "
"reach api.snapcraft.io. If you are an enterprise with a firewall setup you "
"may want to configure a Snap Store proxy."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:485
msgid "Connection to Snap Store failed"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:486
msgid ""
"Your system does not have a connection to the Snap Store. For the best "
"upgrade experience make sure that your system can connect to api.snapcraft."
"io.\n"
"Do you still want to continue with the upgrade?"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:495
msgid "Outdated snapd package"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:496
msgid ""
"Your system does not have the latest version of snapd. Please update the "
"version of snapd on your system to improve the upgrade experience.\n"
"Do you still want to continue with the upgrade?"
msgstr ""

#. now perform direct API calls to the store, requesting size
#. information for each of the snaps needing installation
#: ../DistUpgrade/DistUpgradeQuirks.py:519
msgid "Calculating snap size requirements"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:550
msgid "Processing snap replacements"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:556
#, python-format
msgid "refreshing snap %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:560
#, python-format
msgid "removing snap %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:563
#, python-format
msgid "installing snap %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:592
msgid "PAE not enabled"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:593
msgid ""
"Your system uses a CPU that does not have PAE enabled. Ubuntu only supports "
"non-PAE systems up to Ubuntu 12.04. To upgrade to a later version of Ubuntu, "
"you must enable PAE (if this is possible) see:\n"
"http://help.ubuntu.com/community/EnablingPAE"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:916
msgid "Checking for installed snaps"
msgstr ""

#: ../DistUpgrade/DistUpgradeMain.py:58
msgid "Use the given path to search for a cdrom with upgradable packages"
msgstr "रास्ता (पाथ) प्रदान कर अद्यतन पैकेज वाले सीडीरोम को खोजें"

#: ../DistUpgrade/DistUpgradeMain.py:64
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"अग्रछोड़ का उपयोग करें. वर्तमान में उपलब्ध: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:67
msgid "*DEPRECATED* this option will be ignored"
msgstr ""

#: ../DistUpgrade/DistUpgradeMain.py:70
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr "केवल आंशिक अद्यतन करें (source.list का पुनःलेखन नहीं)"

#: ../DistUpgrade/DistUpgradeMain.py:73
msgid "Disable GNU screen support"
msgstr "GNU स्क्रीन समर्थन असमर्थ करें"

#: ../DistUpgrade/DistUpgradeMain.py:75
msgid "Set datadir"
msgstr "datadir नियत करें"

#: ../DistUpgrade/DistUpgradeMain.py:78
msgid "Upgrade to the development release"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk3.py:146
#: ../DistUpgrade/DistUpgradeViewKDE.py:316
msgid "Fetching is complete"
msgstr "आकर्षण समाप्त हुआ"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:161
#: ../DistUpgrade/DistUpgradeViewKDE.py:332
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "आकर्षित संचिका %li है %li का %sB/s की दर से"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:163
#: ../DistUpgrade/DistUpgradeViewGtk3.py:325
#: ../DistUpgrade/DistUpgradeViewKDE.py:333
#: ../DistUpgrade/DistUpgradeViewKDE.py:483
#, python-format
msgid "About %s remaining"
msgstr "शेष %s के बारे में"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:166
#: ../DistUpgrade/DistUpgradeViewKDE.py:335
#, python-format
msgid "Fetching file %li of %li"
msgstr "आकर्षित संचिका %li है %li का"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk3.py:198
#: ../DistUpgrade/DistUpgradeViewKDE.py:373
msgid "Applying changes"
msgstr "बदलाव लागू कर रहा है"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk3.py:224
#: ../DistUpgrade/DistUpgradeViewKDE.py:386
msgid "dependency problems - leaving unconfigured"
msgstr "निर्भरता समस्ता- विन्यास छोड़ रहा है"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:229
#: ../DistUpgrade/DistUpgradeViewKDE.py:388
#, python-format
msgid "Could not install '%s'"
msgstr "'%s' को संस्थापित नहीं कर सका"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:230
#: ../DistUpgrade/DistUpgradeViewKDE.py:389
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may not be in a working "
"state. Please consider submitting a bug report about it."
msgstr ""
"अद्यतन जारी है लेकिन पैकेज '%s' कार्य करने की स्थिति में नहीं है. कृपया इसके बारे में बग "
"रिपोर्ट भेंजे."

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk3.py:247
#: ../DistUpgrade/DistUpgradeViewKDE.py:411
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"customized विन्यास संचिका को प्रतिस्थापित करें\n"
"'%s'?"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:248
#: ../DistUpgrade/DistUpgradeViewKDE.py:412
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr ""
"यदि आपने नए वर्जन से प्रतिस्थापित किया तो इस विन्यास संचिका में किए गए किसी भी "
"परिवर्तन को आप खो देंगें."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:268
#: ../DistUpgrade/DistUpgradeViewKDE.py:435
msgid "The 'diff' command was not found"
msgstr "'diff' कमांड नहीं पाया गया"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:535
#: ../DistUpgrade/DistUpgradeViewText.py:140
msgid "A fatal error occurred"
msgstr "घातक त्रुटि पायी गई"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:536
msgid ""
"Please report this as a bug (if you haven't already) and include the files /"
"var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"कृपया इसे बग के रुप में रिपोर्ट करें (यदि आप पहले से नहीं किया है) तथा अपने रिपोर्ट में "
"संचिका /var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log को "
"समाहित करें. उन्नयन निष्फल हुआ.\n"
"आपका मुल sources.list को /etc/apt/sources.list.distUpgrade में सहेजा हुआ है."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:566
msgid "Ctrl-c pressed"
msgstr "Ctrl-c दबाएं"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:567
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr ""
"यह प्रक्रिया निष्फल है तथा तंत्र को खंडित स्थिति में छोड़ रहा है. सुनिश्चित करें कि आप इसे "
"करना चाहते हैं?"

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk3.py:706
msgid "To prevent data loss close all open applications and documents."
msgstr "आंकड़े को खोने से बचाने हेतु सभी खुले हुए अनुप्रयोगों एंव दस्तावेजों को बंद करें."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:720
#, python-format
msgid "No longer supported by Canonical (%s)"
msgstr "कैनेनिकल (%s) द्वारा अब समर्थित नहीं है"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:721
#, python-format
msgid "<b>Downgrade (%s)</b>"
msgstr "<b>अवन्नयन (%s)</b>"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:722
#, python-format
msgid "Remove (%s)"
msgstr "हटाएँ (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:723
#, python-format
msgid "No longer needed (%s)"
msgstr "अब और आवश्यकता नहीं है (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:724
#, python-format
msgid "Install (%s)"
msgstr "संस्थापित करें (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:725
#, python-format
msgid "Upgrade (%s)"
msgstr "उन्नयन (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:773
msgid "Cancel"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk3.py:774
#: ../DistUpgrade/DistUpgradeViewKDE.py:939
msgid "Continue"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewKDE.py:447
msgid "Show Difference >>>"
msgstr "अंतर दिखाएं >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:450
msgid "<<< Hide Difference"
msgstr "<<< अंतर छुपाएं"

#: ../DistUpgrade/DistUpgradeViewKDE.py:701
msgid "Error"
msgstr "त्रुटि"

#: ../DistUpgrade/DistUpgradeViewKDE.py:719
#: ../DistUpgrade/DistUpgradeViewKDE.py:960
msgid "&Close"
msgstr "बंद करें (&C)"

#: ../DistUpgrade/DistUpgradeViewKDE.py:753
msgid "Show Terminal >>>"
msgstr "टर्मिनल दिखाएं >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:756
msgid "<<< Hide Terminal"
msgstr "<<< टर्मिनल छुपाएं"

#: ../DistUpgrade/DistUpgradeViewKDE.py:829
msgid "Information"
msgstr "सूचना"

#: ../DistUpgrade/DistUpgradeViewKDE.py:877
msgid "&Start Upgrade"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewKDE.py:879
#: ../DistUpgrade/DistUpgradeViewKDE.py:923
#: ../DistUpgrade/DistUpgradeViewKDE.py:926
#: ../data/gtkbuilder/DistUpgrade.ui.h:7
msgid "Details"
msgstr "विवरण"

#: ../DistUpgrade/DistUpgradeViewKDE.py:901
#, python-format
msgid "No longer supported %s"
msgstr "अब और समर्थित नहीं है (%s)"

#: ../DistUpgrade/DistUpgradeViewKDE.py:903
#, python-format
msgid "Remove %s"
msgstr "%s को हटाएं"

#: ../DistUpgrade/DistUpgradeViewKDE.py:905
#: ../DistUpgrade/DistUpgradeViewText.py:239
#, python-format
msgid "Remove (was auto installed) %s"
msgstr "%s (स्वतः संस्थापित) हटाएं"

#: ../DistUpgrade/DistUpgradeViewKDE.py:907
#, python-format
msgid "Install %s"
msgstr "%s को संस्थापित करें"

#: ../DistUpgrade/DistUpgradeViewKDE.py:909
#, python-format
msgid "Upgrade %s"
msgstr "%s को अद्यतन करें"

#: ../DistUpgrade/DistUpgradeViewKDE.py:956
#: ../DistUpgrade/DistUpgradeViewText.py:292
msgid "Restart required"
msgstr "पुनःआरंभ की जरुरत है"

#: ../DistUpgrade/DistUpgradeViewKDE.py:956
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr "<b><big>अद्यतन पूर्ण करने हेतु तंत्र को पुनःआरंभ करें</big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:959
#: ../data/gtkbuilder/DistUpgrade.ui.h:14
msgid "_Restart Now"
msgstr "अभी पुनःआरंभ करें (_R)"

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:977
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>हो रहे अद्यतन को रद्द करें?</big></b>\n"
"\n"
"यदि आप अद्यतन को रद्द करते हैं तो तंत्र अस्थायी स्थिति में आ जाएगा. आपसे प्रबल सलाह दि "
"जाती है कि अद्यतन को पुनःशुरु करें."

#: ../DistUpgrade/DistUpgradeViewKDE.py:981
msgid "Cancel Upgrade?"
msgstr "अद्यतन रद्द करें?"

#: ../DistUpgrade/DistUpgradeView.py:82
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li दिवस"
msgstr[1] "%li दिवसों"

#: ../DistUpgrade/DistUpgradeView.py:84
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li घंटा"
msgstr[1] "%li घंटे"

#: ../DistUpgrade/DistUpgradeView.py:86
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li मिनट"
msgstr[1] "%li मिनटों"

#: ../DistUpgrade/DistUpgradeView.py:87
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li सेकेण्ड"
msgstr[1] "%li सेकेण्ड"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:103
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s %(str_hours)s"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:121
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:172
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem."
msgstr ""
"इसे डाउनलोड करने में 1मे.ब. DSL संयोजन द्वारा %s तथा 56कि. मोडेम द्वारा %s समय लगेगा."

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:176
#, python-format
msgid "This download will take about %s with your connection. "
msgstr "आपके संयोजन द्वारा इसे डाउनलोड होने में %s समय लगेगा. "

#. Declare these translatable strings from the .ui files here so that
#. xgettext picks them up.
#: ../DistUpgrade/DistUpgradeView.py:270 ../data/gtkbuilder/DistUpgrade.ui.h:22
msgid "Preparing to upgrade"
msgstr "अद्यतन हेतु तैयार हो रहा है"

#: ../DistUpgrade/DistUpgradeView.py:271
msgid "Getting new software channels"
msgstr "नए साफ्टवेयर चैनल प्राप्त कर रहा है"

#: ../DistUpgrade/DistUpgradeView.py:272 ../data/gtkbuilder/DistUpgrade.ui.h:24
msgid "Getting new packages"
msgstr "नया पैकेज प्राप्त कर रहा है"

#: ../DistUpgrade/DistUpgradeView.py:273 ../data/gtkbuilder/DistUpgrade.ui.h:27
msgid "Installing the upgrades"
msgstr "अद्यतन संस्थापित कर रहा है"

#: ../DistUpgrade/DistUpgradeView.py:274 ../data/gtkbuilder/DistUpgrade.ui.h:26
msgid "Cleaning up"
msgstr "साफ कर रहा है"

#: ../DistUpgrade/DistUpgradeView.py:363
#, python-format
msgid ""
"%(amount)d installed package is no longer supported by Canonical. You can "
"still get support from the community."
msgid_plural ""
"%(amount)d installed packages are no longer supported by Canonical. You can "
"still get support from the community."
msgstr[0] ""
"%(amount)d संस्थापित पैकेज अब और कैनेनिकल द्वारा समर्थित नहीं है. आप इस समुदाय से समर्थन "
"प्राप्त कर सकते हैं."
msgstr[1] ""
"%(amount)d संस्थापित पैकेज अब और कैनेनिकल द्वारा समर्थित नहीं है. आप इस समुदाय से समर्थन "
"प्राप्त कर सकते हैं."

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:372
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "%d पैकेज हटाया जा रहा है."
msgstr[1] "%d पैकेज हटाया जा रहा है"

#: ../DistUpgrade/DistUpgradeView.py:377
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "%d नए पैकेज संस्थापित होने जा रहा है."
msgstr[1] "%d नए पैकेज संस्थापित होने जा रहा है."

#: ../DistUpgrade/DistUpgradeView.py:383
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "%d पैकेज उन्नत होने जा रहा है."
msgstr[1] "%d पैकेज उन्नत होने जा रहा है."

#: ../DistUpgrade/DistUpgradeView.py:391
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"आप कुल %s डानलोड करेंगें. "

#: ../DistUpgrade/DistUpgradeView.py:396
msgid ""
"Installing the upgrade can take several hours. Once the download has "
"finished, the process cannot be canceled."
msgstr ""

#: ../DistUpgrade/DistUpgradeView.py:400
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be canceled."
msgstr ""

#: ../DistUpgrade/DistUpgradeView.py:405
msgid "Removing the packages can take several hours. "
msgstr ""

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:410
msgid "The software on this computer is up to date."
msgstr "इस कंप्यूटर पर सॉफ्टवेयर अप टू डेट है |"

#: ../DistUpgrade/DistUpgradeView.py:411
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr "आपके तंत्र हेतु अद्यतन उपलब्ध नहीं है. अद्यतन रद्द हो रहा है."

#: ../DistUpgrade/DistUpgradeView.py:427
msgid "Reboot required"
msgstr "पुनःबूट की आवश्यकता है"

#: ../DistUpgrade/DistUpgradeView.py:428
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr "अद्यतन पूरा हो गया तथा पुनःबूट की जरुरत है. क्या आप इसे करना चाहते हैं?"

#: ../DistUpgrade/DistUpgradeViewText.py:82
#, python-format
msgid "Progress: [%3i%%]"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewText.py:141
msgid ""
"Please report this as a bug and include the files /var/log/dist-upgrade/main."
"log and /var/log/dist-upgrade/apt.log in your report. The upgrade has "
"aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"कृपया इसे बग के रुप रिपोर्ट करें तथा अपने रिपोर्ट में संचिका /var/log/dist-upgrade/main."
"log तथा /var/log/dist-upgrade/apt.log को समाहित करें. उन्नयन निष्फल हो गया.\n"
"आपका मूल sources.list को /etc/apt/sources.list.distUpgrade में सहेजा गया है."

#: ../DistUpgrade/DistUpgradeViewText.py:165
msgid "Aborting"
msgstr "निष्फल हो गया"

#: ../DistUpgrade/DistUpgradeViewText.py:171
msgid "Demoted:\n"
msgstr "पदावनत कियाः\n"

#: ../DistUpgrade/DistUpgradeViewText.py:178
msgid "To continue please press [ENTER]"
msgstr "जारी रखने के लिए कृपया [ENTER] कुंजी दबाएँ"

#: ../DistUpgrade/DistUpgradeViewText.py:215
#: ../DistUpgrade/DistUpgradeViewText.py:253
#: ../DistUpgrade/DistUpgradeViewText.py:261
msgid "Continue [yN] "
msgstr "जारी [yN] "

#: ../DistUpgrade/DistUpgradeViewText.py:215
#: ../DistUpgrade/DistUpgradeViewText.py:253
msgid "Details [d]"
msgstr "विस्तृत [d]"

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeViewText.py:219
#: ../DistUpgrade/DistUpgradeViewText.py:264
msgid "y"
msgstr "हाँ"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:222
#: ../DistUpgrade/DistUpgradeViewText.py:271
msgid "n"
msgstr "न"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:225
msgid "d"
msgstr "विस्तृत"

#: ../DistUpgrade/DistUpgradeViewText.py:230
#, python-format
msgid "No longer supported: %s\n"
msgstr "अब और समर्थित नहीं: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:235
#, python-format
msgid "Remove: %s\n"
msgstr "हटाएं: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:245
#, python-format
msgid "Install: %s\n"
msgstr "संस्थापन: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:250
#, python-format
msgid "Upgrade: %s\n"
msgstr "अद्यतन: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:268
msgid "Continue [Yn] "
msgstr "जारी [Yn] "

#: ../DistUpgrade/DistUpgradeViewText.py:293
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"अद्यतन समाप्त होने के पश्चात पुनःआरंभ की जरुरत होगी.\n"
"यदि आपने  'हां' चुना तो तंत्र पुनःआरंभ हो जाएगा."

#: ../DistUpgrade/GtkProgress.py:71
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr ""
"संचिका डाउनलोड कर रहा है %(current)li को %(total)li में से %(speed)s/s गति से"

#: ../DistUpgrade/GtkProgress.py:77
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "संचिका डाउनलोड कर रहा है %(current)li का %(total)li में से"

#: ../data/gtkbuilder/AcquireProgress.ui.h:1
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:7
msgid "Show progress of individual files"
msgstr "व्यक्तिगत संचिकाओं की प्रगति दिखाएं"

#: ../data/gtkbuilder/DistUpgrade.ui.h:1
msgid "_Cancel Upgrade"
msgstr "अद्यतन निरस्त (_C)"

#: ../data/gtkbuilder/DistUpgrade.ui.h:2
msgid "_Resume Upgrade"
msgstr "अद्यतन दोबारा शुरु करें (_R)"

#: ../data/gtkbuilder/DistUpgrade.ui.h:3
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>अद्यतन को निरस्त करें?</big></b>\n"
"\n"
"यदि आपने अद्यतन को निरस्त कर दिया तो तंत्र अस्वाभाविक स्थिति में आ जाएगा. आपसे पुरजोर "
"आग्रह है कि अद्यतन को पुनःशुरु करें."

#: ../data/gtkbuilder/DistUpgrade.ui.h:6
msgid "_Start Upgrade"
msgstr "उन्नत शुरु करें (_S)"

#: ../data/gtkbuilder/DistUpgrade.ui.h:9
msgid "_Replace"
msgstr "प्रतिस्थापित करें (_R)"

#: ../data/gtkbuilder/DistUpgrade.ui.h:10
msgid "Difference between the files"
msgstr "संचिकाओं में अंतर"

#: ../data/gtkbuilder/DistUpgrade.ui.h:11
msgid "_Report Bug"
msgstr "बग कर रिपोर्ट करें (_R)"

#: ../data/gtkbuilder/DistUpgrade.ui.h:12
msgid "_Continue"
msgstr "जारी रखें (_C)"

#: ../data/gtkbuilder/DistUpgrade.ui.h:13
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>अद्यतन आरंभ करें?</big></b>"

#: ../data/gtkbuilder/DistUpgrade.ui.h:15
msgid ""
"<b><big>Restart the system to complete the upgrade</big></b>\n"
"\n"
"Please save your work before continuing."
msgstr ""
"<b><big>अद्यतन पुरा करने हेतु तंत्र को पुनःआरंभ करे</big></b>\n"
"\n"
"कृपया आगे बढ़ने के पूर्व अपने कार्य को संचित करें."

#: ../data/gtkbuilder/DistUpgrade.ui.h:18
msgid "Distribution Upgrade"
msgstr "वितरण अद्यतन"

#: ../data/gtkbuilder/DistUpgrade.ui.h:20
#, no-c-format
msgid "<b><big>Upgrading Ubuntu to version %s</big></b>"
msgstr ""

#: ../data/gtkbuilder/DistUpgrade.ui.h:21
msgid "    "
msgstr "    "

#: ../data/gtkbuilder/DistUpgrade.ui.h:23
msgid "Setting new software channels"
msgstr "नये साफ्टवेयर चैनल का विन्यास करें"

#: ../data/gtkbuilder/DistUpgrade.ui.h:25
msgid "Restarting the computer"
msgstr "कंप्यूटर को पुनःआरंभ कर रहा है"

#: ../data/gtkbuilder/DistUpgrade.ui.h:28
msgid "Terminal"
msgstr "टर्मिनल"

#: ../data/gtkbuilder/ReleaseNotes.ui.h:2
msgid "_Upgrade"
msgstr "उन्नत (_U)"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:1
msgid "<b>A new version of Ubuntu is available. Would you like to upgrade?</b>"
msgstr "<b>उबुन्टू का एक नया संस्करण उपलब्ध है. क्या आप उन्नत करना चाहेंगे?</b>"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:2
msgid "Don't Upgrade"
msgstr "उन्नत न करें"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:3
msgid "Ask Me Later"
msgstr "बाद में मुझे बताएं"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:4
msgid "Yes, Upgrade Now"
msgstr "हाँ, अब उन्नत करें"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:5
msgid "You have declined to upgrade to the new Ubuntu"
msgstr "आपने नए उबुन्टू में उन्नत करने से अस्वीकार कर दिया है"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:6
msgid ""
"You can upgrade at a later time by opening Software Updater and click on "
"\"Upgrade\"."
msgstr ""

#: ../data/com.ubuntu.release-upgrader.policy.in.h:1
msgid "Perform a release upgrade"
msgstr ""

#: ../data/com.ubuntu.release-upgrader.policy.in.h:2
msgid "To upgrade Ubuntu, you need to authenticate."
msgstr ""

#: ../data/com.ubuntu.release-upgrader.policy.in.h:3
msgid "Perform a partial upgrade"
msgstr ""

#: ../data/com.ubuntu.release-upgrader.policy.in.h:4
msgid "To perform a partial upgrade, you need to authenticate."
msgstr ""

#: ../do-partial-upgrade:75 ../do-release-upgrade:80
msgid "Show version and exit"
msgstr "संस्करण दिखाएँ और बाहर निकलें"

#: ../do-partial-upgrade:78 ../do-release-upgrade:87
msgid "Directory that contains the data files"
msgstr "निर्देशिका जिसमे आंकड़ा संचिका हो"

#: ../do-partial-upgrade:81 ../do-release-upgrade:100
msgid "Run the specified frontend"
msgstr "विशिष्ट अग्रछोड़ को चलाएं"

#: ../do-partial-upgrade:112
msgid "Running partial upgrade"
msgstr "आंशिक कोटि-उन्नयन चलाएं"

#: ../do-release-upgrade:34 ../do-release-upgrade:46
msgid "Downloading the release upgrade tool"
msgstr "प्रकाशित अद्यतन औजार को डाउनलोड करें"

#: ../do-release-upgrade:83
msgid ""
"If using the latest supported release, upgrade to the development release"
msgstr ""

#: ../do-release-upgrade:90 ../check_new_release_gtk.py:181
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr ""
"$distro-proposed द्वारा उन्नयनक्रर्ता का उपयोग कर नविनतम प्रकाशन में कोटि उन्नयन की "
"चेष्टा करें"

#: ../do-release-upgrade:94
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""
"विशिष्ट कोटि-उन्नयन में चलाएं.\n"
"डेक्सटॉप तंत्र का नियमित कोटि उन्नयन हेतु वर्तमान में 'डेक्सटॉप' है तथा सर्वर तेत्र हेतु "
"'सर्वर' समर्थित है."

#: ../do-release-upgrade:103
msgid ""
"Check only if a new distribution release is available and report the result "
"via the exit code"
msgstr ""
"जब केवल नया वितरण प्रकाशन उपलब्ध हो तभी जाँच करें तथा परिणाम को exit कोड द्वारा दें"

#: ../do-release-upgrade:108
msgid ""
"Try the upgrade with third party mirrors and repositories enabled instead of "
"commenting them out."
msgstr ""

#: ../do-release-upgrade:122
msgid "The options --devel-release and --proposed are"
msgstr ""

#: ../do-release-upgrade:123
msgid "mutually exclusive. Please use only one of them."
msgstr ""

#: ../do-release-upgrade:127
msgid "Checking for a new Ubuntu release"
msgstr ""

#: ../do-release-upgrade:139
msgid "Your Ubuntu release is not supported anymore."
msgstr "आपका उबुन्टू प्रकाशन अब और समर्थित नहीं रहा."

#: ../do-release-upgrade:140
msgid ""
"For upgrade information, please visit:\n"
"%(url)s\n"
msgstr ""
"सूचना उन्नयन हेतु, कृपया देखें:\n"
"%(url)s\n"

#: ../do-release-upgrade:147
msgid "In /etc/update-manager/release-upgrades Prompt "
msgstr ""

#: ../do-release-upgrade:148
msgid "is set to never so upgrading is not possible."
msgstr ""

#: ../do-release-upgrade:150
msgid "There is no development version of an LTS available."
msgstr ""

#: ../do-release-upgrade:151
msgid "To upgrade to the latest non-LTS development release "
msgstr ""

#: ../do-release-upgrade:152
msgid "set Prompt=normal in /etc/update-manager/release-upgrades."
msgstr ""

#: ../do-release-upgrade:154
msgid "Upgrades to the development release are only "
msgstr ""

#: ../do-release-upgrade:155
msgid "available from the latest supported release."
msgstr ""

#: ../do-release-upgrade:157
msgid "No new release found."
msgstr ""

#: ../do-release-upgrade:162
msgid "Release upgrade not possible right now"
msgstr "प्रकाशित उन्नयन अब सम्भव नहीं है"

#: ../do-release-upgrade:163
#, c-format
msgid ""
"The release upgrade can not be performed currently, please try again later. "
"The server reported: '%s'"
msgstr ""
"प्रकाशित उन्नयन वर्तमान में नहीं किया जा सकता है, कृपया बाद में पुनः प्रयास करें. सर्वर "
"रिपोर्ट कर रहा है: '%s'"

#: ../do-release-upgrade:169
#, c-format
msgid "New release '%s' available."
msgstr "नया प्रकाशन '%s' उपलब्ध है."

#: ../do-release-upgrade:170
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr "कोटि उन्नयन हेतु 'do-release-upgrade' को चलाएं."

#: ../do-release-upgrade:187
msgid "Please install all available updates for your release before upgrading."
msgstr ""

#: ../do-release-upgrade:203
msgid ""
"You have not rebooted after updating a package which requires a reboot. "
"Please reboot before upgrading."
msgstr ""

#: ../check_new_release_gtk.py:85
msgid "Software Updater"
msgstr ""

#: ../check_new_release_gtk.py:113
#, python-format
msgid "Ubuntu %(version)s Upgrade Available"
msgstr "उबुन्टू %(version)s कोटि-उन्नयन हेतु उपलब्ध है"

#. print("don't upgrade")
#: ../check_new_release_gtk.py:144
#, python-format
msgid "You have declined the upgrade to Ubuntu %s"
msgstr "आपने उबुन्टू %s का कोटि-उन्नयन का अवनति किया है"

#: ../check_new_release_gtk.py:177
msgid "Check if upgrading to the latest devel release is possible"
msgstr "जाँच करे कि नविनतम डेवेल प्रकाशन में कोटि उन्नयन संभव है या नहीं"

#: ../check_new_release_gtk.py:186
msgid "Add debug output"
msgstr "डिबग आउटपुट जोड़ें"

#~ msgid ""
#~ "An unresolvable problem occurred while calculating the upgrade:\n"
#~ "%s\n"
#~ "\n"
#~ " This can be caused by:\n"
#~ " * Upgrading to a pre-release version of Ubuntu\n"
#~ " * Running the current pre-release version of Ubuntu\n"
#~ " * Unofficial software packages not provided by Ubuntu\n"
#~ "\n"
#~ msgstr ""
#~ "अद्यतन की गणना करते समय अनसुलझी समस्या उत्पन्न हो गई है:\n"
#~ "%s\n"
#~ "\n"
#~ " यह इस कारण हो सकता है:\n"
#~ " * उबुन्टू के पूर्व-प्रकाशित वर्जन को अद्यतन करने के कारण\n"
#~ " * वर्तमान में उबुन्टू के पूर्व-प्रकाशित वर्जन के चलने के कारण\n"
#~ " * गैरकार्यालयी साफ्टवेयर पैकेज जो उबुन्टू द्वारा प्रदत्त नहीं है\n"
#~ "\n"

#~ msgid ""
#~ "Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
#~ "desktop or edubuntu-desktop package and it was not possible to detect "
#~ "which version of Ubuntu you are running.\n"
#~ " Please install one of the packages above first using synaptic or apt-get "
#~ "before proceeding."
#~ msgstr ""
#~ "आपके तंत्र में उबुन्टू डेक्सटाप, कुबुन्टू डेक्सटाप, झुबुन्टू डेक्सटाप या एडुबन्टू डेक्सटाप पैकेज नहीं है "
#~ "तथा यह पता करना संभव नहीं है कि आप उबुन्टू का कौन सा वर्जन इस्तेमाल कर रहे हैं |\n"
#~ "कृपया प्रक्रिया के पूर्व उपरोक्त में से किसी एक का पहले संस्थापन सिनेपटिक या apt-get का "
#~ "उपयोग कर करें |"

#~ msgid "Sandbox setup failed"
#~ msgstr "सैंडबाक्स सेटअप असफल"

#~ msgid "It was not possible to create the sandbox environment."
#~ msgstr "सैंडबाक्स वातावरण का निर्माण संभव नहीं है"

#~ msgid "Sandbox mode"
#~ msgstr "सैंडबाक्स विधि"

#~ msgid ""
#~ "Your python install is corrupted. Please fix the '/usr/bin/python' "
#~ "symlink."
#~ msgstr ""
#~ "आपका संस्थापित पायथन विकृत है. कृपया '/usr/bin/python' symlink को ठीक करें |"

#~ msgid "Package 'debsig-verify' is installed"
#~ msgstr "'debsig-verify' पैकेज संस्थापित है"

#~ msgid ""
#~ "The upgrade can not continue with that package installed.\n"
#~ "Please remove it with synaptic or 'apt-get remove debsig-verify' first "
#~ "and run the upgrade again."
#~ msgstr ""
#~ "संस्थापित पैकेज के साथ अद्यतन सम्भव नहीं है |\n"
#~ "कृपया इसे सिनेपटिक या 'apt-get remove debsig-verify' द्वारा पहले हटाएं तब पुनः "
#~ "अद्यतन करें |"

#~ msgid ""
#~ "While scanning your repository information no mirror entry for the "
#~ "upgrade was found. This can happen if you run a internal mirror or if the "
#~ "mirror information is out of date.\n"
#~ "\n"
#~ "Do you want to rewrite your 'sources.list' file anyway? If you choose "
#~ "'Yes' here it will update all '%s' to '%s' entries.\n"
#~ "If you select 'No' the upgrade will cancel."
#~ msgstr ""
#~ "आपके भण्डार सूचना की जाँच करते समय अद्यतन हेतु कोई भी मिरर की प्रविष्टि नहीं पाई गई "
#~ "| यह तभी हो सकता है जब आप आंतरिक मिरर चला रहे हो या फिर मिरर सूचना पुराना हो "
#~ "|\n"
#~ "क्या आप अपने 'sources.list को पुनःलिखना चाहगें? यदि आपने 'हाँ' चुना तो यह सभी "
#~ "'%s' से '%s' तक की प्रविष्टि का अद्यतन हो जाएगा.\n"
#~ "यदि आपने 'नहीं' चुना तो अद्यतन निरस्त हो जाएगा |"

#~ msgid ""
#~ "The upgrade has aborted. The upgrade needs a total of %s free space on "
#~ "disk '%s'. Please free at least an additional %s of disk space on '%s'. "
#~ "Empty your trash and remove temporary packages of former installations "
#~ "using 'sudo apt-get clean'."
#~ msgstr ""
#~ "उन्नयन निष्फल हो गया | उन्नयन हेतु कुल %s मुक्त स्थान डिस्क '%s' पर चाहिए | कृपया कम "
#~ "से कम अतरिक्त %s डिस्क स्थान '%s' पर खाली करें | अपने रद्दी को खाली करें तथा "
#~ "पुर्ववर्ति संस्थापना के अस्थायी पैकेज को 'sudo apt-get clean' का उपयोग कर हटाएँ |"

#~ msgid "evms in use"
#~ msgstr "evms का उपयोग हो रहा है"

#~ msgid ""
#~ "Your system uses the 'evms' volume manager in /proc/mounts. The 'evms' "
#~ "software is no longer supported, please switch it off and run the upgrade "
#~ "again when this is done."
#~ msgstr ""
#~ "आपका तंत्र /proc/mounts मे 'evms' आयतन प्रबंधन का उपयोग कर रहा है. 'evms' "
#~ "साफ्टवेयर अब समर्थित नहीं है, कृपया इसे बंद कर दे तत्पश्चात पुनः अद्यतन करें."

#~ msgid ""
#~ "This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
#~ "version of this driver is available that works with your video card in "
#~ "Ubuntu 10.04 LTS.\n"
#~ "\n"
#~ "Do you want to continue?"
#~ msgstr ""
#~ "यह कंप्यूटर वर्तमान में एऩविडिया का 'एनविडिया' चित्रादि ड्राइवर इस्तेमाल कर रहा है. "
#~ "उबून्टू 10.04 LTS में आपके विडियो कार्ड के साथ कार्य करने हेतु इसका कोई भी संस्करण "
#~ "उपलब्ध नहीं है.\n"
#~ "\n"
#~ "क्या आप आगे बढ़ना चाहते हैं?"

#~ msgid "No i686 CPU"
#~ msgstr "i686 सीपीयु नहीं"

#~ msgid ""
#~ "Your system uses an i586 CPU or a CPU that does not have the 'cmov' "
#~ "extension. All packages were built with optimizations requiring i686 as "
#~ "the minimal architecture. It is not possible to upgrade your system to a "
#~ "new Ubuntu release with this hardware."
#~ msgstr ""
#~ "आपका तंत्र एक i586 सीपीयु या एक सीपीयु की तरह उपयोग हो रहा है जिसमें 'cmov' "
#~ "विस्तारक नहीं हैं. सभी पैकेज i686 के आवश्यकता के अनुकूल न्यूनतम स्थापत्य है. यह सम्भव नही "
#~ "है कि आपके तंत्र का उन्नयन इस हार्डवेयर के साथ नए प्रकाशित उबुन्टू में किया जा सके."

#~ msgid "Sandbox upgrade using aufs"
#~ msgstr "सैंंडबाक्स अद्यतन aufs का उपयोग कर रहा है"

#~ msgid "Test upgrade with a sandbox aufs overlay"
#~ msgstr "सैंडबॉक्स aufs overlay द्वारा उन्नयन करने की जाँच करें"

#~ msgid "No new release found"
#~ msgstr "नया प्रकाशन नहीं पाया गया"
